application/src/app/app.component.ts

```diff
@@ -2,10 +2,12 @@ import { Component } from '@angular/core';
 
 @Component({
   selector: 'vote-root',
-  template: `    
+  template: `
     <!--The content below is only a placeholder and can be replaced.-->
-    <div class="container">
-      <div class="col-md-1 col-lg-12">Welcome to One Vote!!!</div>
+    <div class="container col-sm-12 col-lg-3">
+      <div>Welcome to One Vote!!!</div>
+      <p>This is your Login Page</p>
+      <router-outlet></router-outlet>
```

Comment:
should me put this code into a .html file, and put the path here?

Reply:
nm, i figured. Since we wouldn't reuse this piece of code , so i guess its fine to put here

Reply:
This is the root component of the app, which will be put into index.html.

Reply:
There are two ways to write html for a component, inline html, or link to a html file. This is the preferred way.